version.go

```diff
@@ -1,3 +1,3 @@
 package main
 
-const VERSION = "1.4.3"
```

Comment:
Do you think we need to increase the version when we modify the pipeline?

Reply:
Not really, but given that it will publish it when merging it to master, we considered (with @bmcfeely ) to just increase the patch version. I could keep the 1.4.3 version, but then I think it would be republished when merging anyway. 🤔

Reply:
I think if you maintain the version. The pipeline will fail... but it will fail for the right reasons.